index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML + basic JavaScript Capstone</title>
+
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="styles.css">
+
+    <!-- font family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet">
+
+    <!-- boostrap -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous">
+    <!-- icons -->
+    <link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+  </head>
+
+
+  <body>
+    <header class="header">
+      <nav>
+        <div class="navbar navbar-expand-lg navbar-dark bg-dark navbar-social">
+          <div class="container-fluid ">
+            <div class="collapse navbar-collapse justify-content-end
+              navbar-nav-social" id="navbarNav">
+              <ul class="navbar-nav nav-bar-items-social">
+                <li class="nav-item">
+                  <a class="nav-link active nav-icon" aria-current="page"
+                    href="#"><i class="bi bi-facebook"></i></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link active" href="#"><i class="bi bi-twitter"></i></a>
+                </li>
```

Comment:
- Kindly, the social icons and language sections are not part of the  navigation so please make them outside  the `nav` tag for semantic reasons 👍
- Kindly, you have given unnecessary space between the social icons and language please update it to have a better design and follow the given design 👍
### your design 👇
![2023-04-20_23-06](https://user-images.githubusercontent.com/52010789/233476721-5b7881c0-10ec-4db3-9f56-e591ae49b89f.png)
### Given design 👇
![2023-04-20_23-09](https://user-images.githubusercontent.com/52010789/233476913-52c198e1-e625-40a9-a7f0-13edca00d79f.png)